pytrip/cube.py

```diff
@@ -394,7 +394,7 @@ def write_trip_header(self, path):
             output_str += "slice_no  position  thickness  gantry_tilt\n"
             for i, item in enumerate(self.slice_pos):
                 output_str += "{:3d}{:16.4f}{:13.4f}{:14.4f}\n".format(i + 1, item,
-                                                                       self.slice_distance, 0)  # 0 gantry tilt
+                                                                       self.slice_thickness, 0)  # 0 gantry tilt
```

Comment:
I made to hed+ctx files with either slice_distance or slice_thickness using the overlapping slice DICOM-CT i have. Made a proton plan on both of them, and TRiP mapped it correctly in both cases. So I prefer passing slice_thickness here as it is more correct, and slice_distance can be derived anyway from the z_position.